test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -700,6 +700,249 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
     }
   }
 
+  void test_as_scalars_vs_as_vector() {
+    if (!any_vector<T0, T1, T2, T3, T4, T5>::value) {
```

Comment:
Why/how do all types being scalars signal there is nothing to test?

Reply:
If there are no vector types tested, then this is not a test of a vectorization.

Reply:
(And the test here is checking that vectorization works)